src/components/forms/LocationSelect/tests/index.test.tsx

```diff
@@ -0,0 +1,10 @@
+import { mount, shallow } from 'enzyme';
+import React from 'react';
+import { LocationSelect } from '..';
+
+describe('components/forms/LocationSelect', () => {
+  it('renders without crashing', () => {
+    shallow(<LocationSelect />);
+    mount(<LocationSelect />);
+  });
+});
```

Comment:
Can you write better tests please?  In this case you just need to test that the correct props are passed to the JurisdictionSelect component.